Web/Edubase.Web.UI/Web.config

```diff
@@ -274,6 +274,18 @@
 <!-- <add key="HttpAuthModule.Credentials" value="" /> -->
     <add key="HttpAuthModule.IgnoreIPAddresses" value="127.0.0.1;::1" />
 
+
+
+    <!-- TIMEOUT SETTINGS -->
+
+    <add key="FBService_Timeout" value="2" />
+    <add key="HttpClient_Timeout" value="30" />
```

Comment:
Not sure about this one but probably not a good idea to hold up the story as it can be configured later.

Reply:
1 - yeah the 2 seconds and 30 seconds were suggestions by Scott left on the ticket ....bit of trial and error